lib/ui/window.dart

```diff
@@ -6,7 +6,7 @@ part of dart.ui;
 /// A view into which a Flutter [Scene] is drawn.
 ///
 /// Each [FlutterView] has its own layer tree that is rendered into an area
-/// inside of a [FlutterWindow] whenever [render] is called with a [Scene].
+/// inside of a [SingletonFlutterWindow] whenever [render] is called with a [Scene].
```

Comment:
nit: let's reword this paragraph without mentioning SingletonFlutterWindow or FlutterWindow. It doesn't make super sense, FlutterViews can stand alone without any window.